packages/flutter/lib/src/widgets/scrollable.dart

```diff
@@ -456,14 +455,15 @@ class ScrollableState<T extends Scrollable> extends State<T> {
     updateGestureDetector();
   }
 
-  /// Fling the scroll offset with the given velocity.
+  /// Fling the scroll offset with the given velocity in logical pixels/second.
   ///
   /// Calling this function starts a physics-based animation of the scroll
   /// offset with the given value as the initial velocity. The physics
-  /// simulation used is determined by the scroll behavior.
+  /// simulation is determined by the scroll behavior.
   Future<Null> fling(double scrollVelocity) {
```

Comment:
Velocity has X and Y components but this velocity is defined to be in the direction of the scrollable's axis.

It might be nice to carry a Velocity type around anyway; maybe add a Velocity.along(Axis) method.  Eventually, in classes like ScrollBehavior that don't have an axis, I think we'd still need a velocity scalar.